.travis.yml

```diff
@@ -5,8 +5,6 @@ go:
   - "1.10"
   - "1.11"
   - "tip"
-install:
-  - mkdir -p $HOME/gopath/src/gopkg.in/gemnasium
-  - mv $HOME/gopath/src/github.com/gemnasium/logrus-graylog-hook $HOME/gopath/src/gopkg.in/gemnasium/logrus-graylog-hook.v2
-  - cd $HOME/gopath/src/gopkg.in/gemnasium/logrus-graylog-hook.v2
-  - go get -t
+
+env:
+  - GO111MODULE=on
```

Comment:
by enabled this  default travis "go get" will fetch the correct dependencies from go.mod

It seems that this is enough